timemachine/md/moves.py

```diff
@@ -103,6 +96,31 @@ def n_proposed_by_move(self) -> List[int]:
         return [m._n_proposed for m in self.moves]
 
 
+class MixtureOfMoves(CompoundMove[_State]):
+    """Apply a single move uniformly selected from a list"""
+
+    def __init__(self, moves: Sequence[MonteCarloMove[_State]]):
+        self.moves = moves
+
+    def move(self, x: _State) -> _State:
+        idx = np.random.choice(len(self.moves))
```

Comment:
Seems preferable to be able to seed this

Reply:
Will address along with https://github.com/proteneer/timemachine/pull/1128#discussion_r1310314713